docs/data_collection.rst

```diff
@@ -160,7 +161,8 @@ elements that can be tracked this way include:
 Google Analytics
 ----------------
 
-Some Mozilla websites use Google Analytics to collect data about user behavior
-so that we can improve our sites. Neither the Snippets Service nor the code on
-about:home use Google Analytics directly, and Google Analytics is never loaded
-within about:home.
+The `snippets statistics server <https://google.com>`_ may proxy data to Google
+Analytics, with stripped IP information and with a unique to every request,
+randomly generated UID. Google Analytics is never loaded within about:home. Some
+Mozilla websites use Google Analytics to collect data about user behavior so
+that we can improve our sites.
```

Comment:
Suggest re-wording:

The `snippets statistics server <https://google.com>`_ may proxy data  to Google Analytics, with stripped IP information and with a randomly generated UID unique to every request.